server/polar/organization/endpoints.py

```diff
@@ -214,3 +220,64 @@ async def members(
         items=[OrganizationMember.model_validate(m) for m in members],
         pagination=Pagination(total_count=len(members), max_page=1),
     )
+
+
+@router.post(
+    "/{id}/members/invite",
+    response_model=OrganizationMember,
+    tags=[APITag.private],
+    status_code=201,
+)
+async def invite_member(
+    id: OrganizationID,
+    invite_body: OrganizationMemberInvite,
+    auth_subject: auth.OrganizationsWrite,
+    session: AsyncSession = Depends(get_db_session),
+) -> OrganizationMember:
+    """Invite a user to join an organization."""
+    organization = await organization_service.get(session, auth_subject, id)
+
+    if organization is None:
+        raise ResourceNotFound()
+
+    # Get or create user by email
+    user, _ = await user_service.get_by_email_or_create(session, invite_body.email)
+
+    # Add user to organization
+    await organization_service.add_user(session, organization, user)
+
+    # Get the inviter's email (from auth subject)
+    inviter_email = (
+        auth_subject.subject.email
+        if hasattr(auth_subject.subject, "email")
+        else "A team member"
+    )
+
+    # Send invitation email
+    body = render_email_template(
+        "organization_invite",
+        {
+            "organization_name": organization.name,
+            "inviter_email": inviter_email,
```

Comment:
`mypy` is failing here because `EmailStr` is not considered as a strict `str`. So we should `str(inviter_email)`.